scripts/apps/monitoring/directives/MonitoringGroup.ts

```diff
@@ -710,7 +711,17 @@ export function MonitoringGroup(
              * return {promise} list of items
              */
             function apiquery(searchCriteria, applyProjections) {
+                const personalGroup = ['personal', 'sent'];
                 var provider = 'search';
+                const personalSectionIds = [];
```

Comment:
You can simplify this

`const personalSectionIds = getExtensionSections().map(({id}) => id);`